local_land_charges_api_stub/documentation/local-land-charges-api.yml

```diff
@@ -628,62 +664,82 @@ definitions:
+    description: The address of the applicant. Valid only when 'charge-type' is 'Light Obstruction Notice'
     $ref: '#/definitions/applicant-address-structure'
   servient-land-interest-description:
-    description: "The capacity in which the applicant holds the land over which the acquisition of the right to light is prevented (The Servient Land). Valid only when 'charge-type' is 'Light Obstruction Notice'"
+    description: The capacity in which the applicant holds the land over which the acquisition of the right to light is prevented (The Servient Land). Valid only when 'charge-type' is 'Light Obstruction Notice'
     type: string
     enum: ["Owner", "Tenant", "Lender"]
   structure-position-and-dimension:
-    description: "The position and dimension of a notional structure which if erected would block the light to the dominant land. This structure is notionally erected on the servient land (the dominant land is the subject of the charge). Valid only when 'charge-type' is 'Light Obstruction Notice'"
-    type: string
+    description: The position and dimension of a notional structure which if erected would block the light to the dominant land. This structure is notionally erected on the servient land (the dominant land is the subject of the charge). Valid only when 'charge-type' is 'Light Obstruction Notice'
+    type: object
     minLength: 1
-    example: "'All of the boundaries of the servient land','The land marked by a red line between points A and B on the attached plan'"
+    properties:
+      height:
+        type: string
+        pattern: '^(Unlimited height)|(\d+(\.\d+)?)$'
+      units:
+        type: string
+        description: The unit of measurement of the structure height. Must be provided if height is not 'Unlimited height'
+        enum:
+          - Metres
+          - Feet
+      extent-covered:
+        type: string
+        enum:
+          - All of the extent
+          - Part of the extent
+      part-explanatory-text:
+        type: string
+        description: The part of the extent covered by the structure. Must be provided if extent-covered is 'Part of the extent'
+    required:
+      - height
+      - extent-covered
   tribunal-definitive-certificate-date:
-    description: "The date the definitive Lands Tribunal certificate was issued by the Lands Tribunal. Valid only when 'charge-type' is 'Light Obstruction Notice' and documents-filed includes a definitive-land-certificate"
+    description: The date the definitive Lands Tribunal certificate was issued by the Lands Tribunal. Valid only when 'charge-type' is 'Light Obstruction Notice'
     type: string
     pattern: '^\d{4}-\d{2}-\d{2}$'
     format: date
     example: '2017-01-04'
   documents-filed:
-    description: "The documents lodged to support the LON registration. Valid only when 'charge-type' is 'Light Obstruction Notice'"
-    enum: ["form-a", "colour-plan", "definitive-certificate"]
```

Comment:
This should be left as is. LONs can't be added or amended via the api, only viewed, and only the descriptions of the available documents are shown not the actual locations. However it needs temporary-lon-certificate added?

Reply:
This has been changed to a $ref: to the actual object (the enum's not correct)

Reply:
Per discussion, fixed